supplementary/alica_ros1/alica_ros_turtlesim/etc/plans/TeleportToRandomPosition.pml

```diff
@@ -0,0 +1,265 @@
+{
+  "blackboard": [
+    {
+      "access": "protected",
+      "comment": "",
+      "id": 1417221173347283069,
+      "key": "xmin",
+      "type": "double"
+    },
+    {
+      "access": "protected",
+      "comment": "",
+      "id": 134046690438663400,
+      "key": "xmax",
+      "type": "double"
+    },
+    {
+      "access": "protected",
+      "comment": "",
+      "id": 563252773862417130,
+      "key": "ymin",
+      "type": "double"
+    },
+    {
+      "access": "protected",
+      "comment": "",
+      "id": 2382626652775529582,
+      "key": "ymax",
+      "type": "double"
+    },
+    {
+      "access": "protected",
+      "comment": "",
+      "id": 598868303244083964,
+      "key": "x",
+      "type": "double"
+    },
+    {
+      "access": "protected",
+      "comment": "",
+      "id": 2712475924298828232,
+      "key": "y",
+      "type": "double"
+    }
+  ],
+  "comment": "",
+  "entryPoints": [
+    {
+      "comment": "",
+      "id": 716543537979003414,
+      "isDynamic": false,
+      "maxCardinality": 2147483647,
+      "minCardinality": 0,
+      "name": "",
+      "plan": 847199748749956244,
+      "positionWeb": {
+        "x": 271.5532139093783,
+        "y": 303.9905163329821
+      },
+      "state": 815756385850055878,
+      "successRequired": false,
+      "task": "TaskRepository.tsk#3310236980587704776"
+    }
+  ],
+  "frequency": 0,
+  "id": 847199748749956244,
+  "inheritBlackboard": false,
+  "libraryName": "libalica-turtlesim",
+  "masterPlan": false,
+  "name": "TeleportToRandomPosition",
+  "preCondition": null,
+  "relativeDirectory": "",
+  "runtimeCondition": null,
+  "states": [
+    {
+      "comment": "",
+      "confAbstractPlanWrappers": [
+        {
+          "abstractPlan": "behaviours/GenerateRandom.beh#3356053372497284615",
+          "comment": "",
+          "configuration": null,
+          "id": 172292232628300347,
+          "keyMapping": {
+            "input": [
+              {
+                "childKey": "min",
+                "parentKey": "xmin",
+                "value": null
+              },
+              {
+                "childKey": "max",
+                "parentKey": "xmax",
+                "value": null
+              }
+            ],
+            "output": [
+              {
+                "childKey": "value",
+                "parentKey": "x"
+              }
+            ]
+          },
+          "name": ""
+        },
+        {
+          "abstractPlan": "behaviours/GenerateRandom.beh#3356053372497284615",
```

Comment:
don't we need to separate these into 2 states for now?

Reply:
you're right, probably x/y for spawn are always the same now

Reply:
I regenerated the etc folder with the updated dev image of the designer, so the folder structure of etc should also be sorted